app/Main.hs

```diff
@@ -62,6 +69,27 @@ dotER conf er = graph' $ do
         label    = A.Label $ A.HtmlLabel $ H.Text $ withLabelFmt " %s " optss []
     edge (entity1 r) (entity2 r) [label, l1, l2]
 
+-- | Converts an entire ER-diagram from an ER file into a GraphViz graph.
+dotERNoHtml:: Config -> ER -> G.DotGraph L.Text
+dotERNoHtml conf er = graph' $ do
+  graphAttrs (graphTitle $ title er)
+  graphAttrs [ A.RankDir A.FromLeft
+             , A.Splines $ fromMaybe (fromJust . edgeType $ defaultConfig) (edgeType conf)
+             ]
+  nodeAttrs [shape Record, A.RankDir $ A.FromTop]  
```

Comment:
This seems to be the only line which differs from the function: **Main::dotER**. It would be important to refactor the ER-diagram generator to avoid code duplication.

Reply:
There's also another minor difference (64 vs 85), but I agree that this should be refactored. What I am not sure of
is exactly how. I thought about adding where clauses in dotER that would handle the switching:
```suggestion
  -nodeAttrs [shape Record, A.RankDir $ A.FromTop]
  +nodeAttrs nodeAttributes
  (...)
  - node (name e) [toLabel (htmlEntity e)]
  + node (name e) [toLabel (entityFmt e)]
  where nodeAttributes 
             | nohtml config = [shape Record, A.RankDir $ A.FromTop]  
             | otherwise = [shape PlainText]
         entityFmt 
             | nohtml config = dotEntity
             | otherwise = htmlEntity
```

Reply:
I have added a new commit which _"tries"_ to solve the refactoring problem